deploy/role.yaml

```diff
@@ -41,15 +54,27 @@ rules:
   verbs:
   - update
 - apiGroups:
-  - pulpproject.org
+  - ""
   resources:
-  - '*'
+  - pods
   verbs:
-  - '*'
+  - get
 - apiGroups:
-  - ""
+  - apps
+  resources:
+  - replicasets
+  - deployments
+  verbs:
+  - get
+- apiGroups:
+  - pulpproject.org
   resources:
-  - nodes
+  - '*'
   verbs:
+  - create
+  - delete
   - get
   - list
+  - patch
+  - update
+  - watch
```

Comment:
This is great! Thank you for narrowing down the permissions.

Reply:
It came from `v0.18.2` template